proto/time.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
```

Comment:
I think a top-level proto package could be a bit misdirecting to a contributor.

Can we just toss this type in one of the new packages instead?

Reply:
it could go in tailnet.proto.  I was trying to avoid import awkwardness later if we need a timestamp later in some unrelated API.